src/sbus/codegen/sbus_Generator.py

```diff
@@ -18,9 +18,10 @@
 #   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-from sbus_Invoker import *
+from sbus_Invoker import Invoker, InvokerCaller, InvokerKeygen, InvokerArgumentType
```

Comment:
This also gave me some headaches some months ago I stopped trying to fix the issues.

Reply:
Reverted